### PR TITLE
fix(warnings): Add actionable URL for system warnings popup

### LINKED ISSUE #804: "There is 1 issue with your system configuration." is not actionable
## Version Information

Version: Sentry 21.1.0.dev0f96907d

## Description

image

I upgraded Sentry, now I get this message.
It's good to know that there's an issue, but it's quite useless as it is.
- The message is not clickable
- The interface doesn't show the actual error anywhere
- It doesn't offer any suggestion where to look for the cause

## Steps to Reproduce

1. Upgrade sentry to get this error
2. That's what I would like to know :) I'll probably have to dig in the logs or something so I'll solve it.

## Logs

Not relevant. My issue isn't the problem that's causing the error, only the useless error message.

### PR DESCRIPTION
**Pull request by BYK**
Fixes getsentry/onpremise#804

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]
You're going to error on the `str.format` call because there's not enough placeholders.

**Comment by BYK** [comment]
>You're going to error on the str.format call because there's not enough placeholders.

Didn't touch there and it seems to have all 3 arguments and placeholders?

**Comment by mattrobenolt** [comment]
Oh. My bad. I see.

File: src/sentry/status_checks/warnings.py [hunk 1]
```diff
@@ -17,6 +21,10 @@ def check(self):
                         "issues" if len(self.__warning_set) > 1 else "issue",
                     ),
                     severity=Problem.SEVERITY_WARNING,
+                    # We need this manual URL building as this page is moved to react
+                    # and only the top-level entrypoint is defined and addressable in
+                    # our backend Django app.
+                    url=urljoin(reverse("sentry-admin-overview"), "status/warnings/"),
```
Comment (BYK) [inline src/sentry/status_checks/warnings.py:27]: @EvanPurkhiser @wedamija is this fine or should I just put a literal string here (feels worse to do that)

Comment (EvanPurkhiser) [reply src/sentry/status_checks/warnings.py:27]: nah this is better IMO

Comment (BYK) [reply src/sentry/status_checks/warnings.py:27]: Then setting up auto-merge, here we go... 🚀

Comment (BYK) [reply src/sentry/status_checks/warnings.py:27]: @EvanPurkhiser I wonder if we can make these links not treated as external links (which is the default now)? Is it safe to pass `ReactRouter` related stuff from here? I can pass it to `to=` but that feels unsafe.

**Pull request merged**